application/views/admin/revisions/partials/revision_header.php

```diff
@@ -80,14 +81,16 @@
 	</div>
 	<div class="modal-body">
 	  <p><?php echo __('revisions.modals.live_warning'); ?></p>
-	  <p><?php echo __('modals.confirm_body'); ?></p>
+      <p><?php echo __('modals.confirm_body'); ?></p>
 	</div>
 	<div class="modal-footer">
 	    <a data-dismiss="modal" href="#" class="btn"><?php echo __("revisions.cancel"); ?></a>
    		<a class="btn btn-danger yes_action"><?php echo __("revisions.make_live"); ?></a>
 	</div>
 </div>
+
 <div class="modal hide fade" id="send_for_editing">
+	<form action="#" method="post">
```

Comment:
make this a form that will submit via js instead of just an ok link to a GET url ffs